common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -39,6 +39,29 @@ div.problem-progress {
   font-size: em(16);
 }
 
+div.problem-timer {
+  display: none;
+  background: $white;
+  border: 1px solid #dedede;
+  color: $green;
+  font-size: 14px;
+  font-weight: bold;
+  margin-top: 14px;
+  margin-bottom: 14px;
+  padding: 7px 21px;
+  z-index: 9999;
```

Comment:
I thought there was a platform convention for managing/specifying z-indexes (a mixin, presumably); @caesar2164 do you know?